test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -0,0 +1,627 @@
+---------------------------------------------------------------------------------------------------
+-- RC common module for AUDIO, LIGHT, HMI_SETTINGS modules
+---------------------------------------------------------------------------------------------------
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application2.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
```

Comment:
Should just update the default version in `config.lua`, rather than setting it here.

Reply:
config.lua this is ATF files, so to run scripts without any changed we should update ATF, so i think that this is not good idea, scripts can by itself config all needed configuration for it. These scripts need 5.0 version, another scripts will require 3.0 version, so required version must be defined in scripts.

Reply:
I think that test scripts that specifically require older versions should specify this, not any tests which are just testing against the latest spec.

Reply:
okay, but these changes for ATF are not related to current proposal, so for now we can leave such workaround and will provide removing majorVersion from scripts and changes for config.lua as fixes for mobile versioning feature.